binr/rabin2/rabin2.c

```diff
@@ -590,9 +590,14 @@ int main(int argc, char **argv) {
 		}
 		r_lib_opendir (l, homeplugindir);
 		free (homeplugindir);
-		r_lib_opendir (l, R2_LIBDIR "/radare2/" R2_VERSION);
-		r_lib_opendir (l, R2_LIBDIR "/radare2-extras/" R2_VERSION);
-		r_lib_opendir (l, R2_LIBDIR "/radare2-bindings/" R2_VERSION);
+		const char *dirPrefix = r_sys_prefix (NULL);
+		char folder[1024];
```

Comment:
use r_str_newf. paths can be bigger than 1024 in some systems. in fact in hurd paths can be infinite

Reply:
Actually it is copy/paste from https://github.com/radare/radare2/blob/master/libr/asm/asm.c#L305. But ok I will replace it with `r_str_r2_prefix()`.